File: WOT-iOS/WOTData/WOTData/model/subclasses/VehicleprofileAmmoDamage+CoreDataProperties.m
//
//  VehicleprofileAmmoDamage+CoreDataProperties.m
//  WOTData
//
//  Created on 8/28/18.
//  Copyright © 2018. All rights reserved.
//
//

#import "VehicleprofileAmmoDamage+CoreDataProperties.h"

@implementation VehicleprofileAmmoDamage (CoreDataProperties)

+ (NSFetchRequest<VehicleprofileAmmoDamage *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"VehicleprofileAmmoDamage"];
}

@dynamic avg_value;
@dynamic max_value;
@dynamic min_value;
@dynamic vehicleprofileAmmo;

@end

File: WOT-iOS/WOTApi/source/model/autogenerated/VehicleprofileAmmoDamage+CoreDataProperties.swift
//
//  VehicleprofileAmmoDamage+CoreDataProperties.swift
//  WOTData
//
//  Created by Pavel Yeshchyk on 5/1/20.
//  Copyright © 2020 Pavel Yeshchyk. All rights reserved.
//
//

import CoreData

public extension VehicleprofileAmmoDamage {
    @nonobjc class func fetchRequest() -> NSFetchRequest<VehicleprofileAmmoDamage> {
        return NSFetchRequest<VehicleprofileAmmoDamage>(entityName: "VehicleprofileAmmoDamage")
    }

    @NSManaged var avg_value: NSDecimalNumber?
    @NSManaged var max_value: NSDecimalNumber?
    @NSManaged var min_value: NSDecimalNumber?
    @NSManaged var vehicleprofileAmmo: VehicleprofileAmmo?
}
